ironfish/src/webApi.ts

```diff
@@ -158,7 +158,18 @@ export class WebApi {
     return await axios
       .get<ApiUser>(`${this.host}/users/find`, options)
       .then((r) => r.data)
-      .catch(() => null)
+      .catch((error: AxiosError<{ code: string; message?: string }>) => {
+        if (error.response) {
+          const { status } = error.response
+          if (status >= 500) {
+            throw new Error(`API status code: ${status}`)
```

Comment:
Let's make this a bit more descriptive

```suggestion
            throw new Error(`API request to fetch user '${graffiti}' failed with status code: ${status}`)
```